.ci/deps.opam.sh

```diff
@@ -11,6 +11,10 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   opam init --y
   opam update
 
+  opam pin add --yes --no-action yojson 1.3.3
```

Comment:
Add `# See <URL>` above this line, like I did below, and put the two commands together as they are part of the same fix.

Reply:
Ok